src/network/netdev/macvlan.c

```diff
@@ -107,6 +152,7 @@ static void macvlan_init(NetDev *netdev) {
 
         m->mode = _NETDEV_MACVLAN_MODE_INVALID;
         m->bc_queue_length = UINT32_MAX;
+        m->bc_queue_threshold = INT32_MAX;
```

Comment:
As mentioned above, INT32_MAX is a valid value.
Please introduce a simple enum something like the following and use it:
```c
BC_QUEUE_THRESHOLD_UNDEF = INT32_MIN,
BC_QUEUE_THRESHOLD_DISABLE = -1,
```